Plugins/Creature/NWScript/nwnx_creature.nss

```diff
@@ -61,6 +61,8 @@ const int NWNX_CREATURE_PROJECTILE_VFX_SONIC        = 5;
 const int NWNX_CREATURE_PROJECTILE_VFX_RANDOM       = 6; ///< Random Elemental VFX
 /// @}
 
+const int ABILITY_NONE = 6;
```

Comment:
Would prefer this to be prefixed with `NWNX_CREATURE_` + a note in the function desc.